### PR TITLE
[aws-utils] Parameterize encrypted text for test so different environments can us…

### PR DESCRIPTION
**Pull request by janedotx**
…e different AWS credentials.

## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance

### CODE REVIEW

File: aws-lambda/aws-utils/tests/test.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
 import * as assert from 'assert'
 import { decrypt } from '../index'
+const { env } = process
```
Comment (mdurling) [inline aws-lambda/aws-utils/tests/test.ts]: How about `const { ENCRYPTED_TEXT } = process.env`?

**Comment by avatarneil** [comment]
Mr. Jenkins, run tests pls

File: aws-lambda/aws-utils/tests/test.ts [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 describe('Test AWS utils', () => {
   it('can decrypt cipher text', async () => {
-    const plainText = await decrypt(
-      'AQICAHhAYaddUZOibOvj/rRTUvSdxqwlaiZHvOM4i7XV4DEr5wFz' +
-        'KI17osSCxRlGVxdSWP8/AAAAZjBkBgkqhkiG9w0BBwagVzBVAgEA' +
-        'MFAGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMt31oGEx/Hh78' +
-        'ky9YAgEQgCMbMNiEQgOFEM66S5GncBdexWG+jCVgY48oAMSZe051dfeVvg=='
-    )
-    assert.deepEqual(plainText, 'password')
+   if (!env.ENCRYPTED_TEXT) {
```
Comment (avatarneil) [inline aws-lambda/aws-utils/tests/test.ts]: Can we set a default for `ENCREYPTED_TEXT` somewhere?

Comment (janedotx) [reply aws-lambda/aws-utils/tests/test.ts]: that's a really good idea, i should've thought of that

**Comment by janedotx** [comment]
hey mr jenkins run me a test

**Comment by avatarneil** [comment]
> hey mr jenkins run me a test

Hehe that won't work, gotta say something that matches `.*(re)?run tests.*`. Something like, run tests my good friend Mr. Jenkins!

**Pull request merged**